test/browser/features/BrowserBasedTests/HappyPath/EEAIdentityCard/F2FNICardEEANoAddressEmail.feature

```diff
@@ -0,0 +1,40 @@
+@mock-api:f2f-f2f-success @success @browser
+Feature: Document Selection - National Identity Card EEA E2E Journey (Happy Path)
+
+    Background:
+        Given An EEA Identity Card User is using the system
+        When they have provided their details
+        Then they should be redirected to the Landing Page
+
+        Given the user wants to progress to the next step of the journey
+        When the user clicks the continue button on the Landing Page
+        Then the user is routed to the next screen in the journey PhotoId Selection
+
+        Given the EEA National Identity Card option is selected
+        When the user clicks the PhotoId continue button with EEA National Identity Card selected
+        Then the user is routed to the EEA Has Expiry Entry Screen
+
+        When the user selects yes on the EEA identity expiry date page
+        Then the user is routed to the next screen in the EEA National Identity journey - EEA National Identity Card details
+
+        Given the date entered is within accepted National Identity Card EEA expiration window
+        When the user clicks the continue button on the National Identity Card EEA Page
+        Then the user is routed from NI Card EEA Details to the address check page
+
+        Given the user selects My identity card does not have my address on it
+        When the user clicks continue on the EEA Identity Card address check page
+        Then they are routed to the EEA ID country code selection screen
+
+        Given the user is on the NI Card EEA Country Code Selection screen
+        When the user selects an EEA country code
+        Then the user is routed from NI Card EEA Country to Branch Finder Screen
+
+        Given the postcode entered is valid
+        When the user clicks the continue button on the find Post Office branch page
+        Then the user is routed to the Select Location page showing 5 nearest POs
+
+    Scenario: NI Card EEA - E2E Journey (Happy Path)
```

Comment:
Can we update the scenario names to be more explicit like the file names as these are what will appear in the test report